x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -3782,6 +3782,8 @@
     "xpack.apm.formatters.millisTimeUnitLabel": "ms",
     "xpack.apm.formatters.requestsPerMinLabel": "rpm",
     "xpack.apm.formatters.secondsTimeUnitLabel": "s",
+    "xpack.apm.formatters.minutesTimeUnitLabel": "m",
+    "xpack.apm.formatters.hoursTimeUnitLabel": "h",
```

Comment:
The transaction files are auto-generated so if you created this and the Japanese by hand you can just discard them again.